File: ts/iq/waffle~get~encryption_metadata+fbid+timestamp.ts
/** <iq type="get" xmlns="waffle">. Replies with one of: Success, Error. */
export function buildWaffleRefreshAccessTokens(smaxId: number): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'waffle', smax_id: smaxId, to: 's.whatsapp.net', type: 'get' }, content: [
      { tag: 'encryption_metadata', attrs: {  } },
      { tag: 'timestamp', attrs: {  } },
      { tag: 'fbid', attrs: {  } },
    ] }
}

File: rust/iq/waffle~get~encryption_metadata+fbid+timestamp.rs
/// <iq type="get" xmlns="waffle">. Replies with one of: Success, Error.
pub fn build_waffle_refresh_access_tokens(smaxId: i64) -> Node {
    NodeBuilder::new("iq")
        .attr("xmlns", "waffle")
        .attr("smax_id", smaxId)
        .attr("to", "s.whatsapp.net")
        .attr("type", "get")
        .child(NodeBuilder::new("encryption_metadata").build())
        .child(NodeBuilder::new("timestamp").build())
        .child(NodeBuilder::new("fbid").build())
        .build()
}